easybuild/easyconfigs/n/numactl/numactl-2.0.10-GCC-4.9.2.eb

```diff
@@ -11,6 +13,13 @@ The libnuma library provides convenient ways for you to add NUMA memory policies
 sources = [SOURCE_TAR_GZ]
 source_urls = ['ftp://oss.sgi.com/www/projects/libnuma/download/']
 
+builddependencies = [
+    ('Automake', '1.15'),
+    ('libtool', '2.4.2'),
+]
+
+preconfigopts = './autogen.sh && '
+
 toolchain = {'name': 'GCC', 'version': '4.9.2'}
```

Comment:
nitpicking: we usually place the `toolchain` line above the `sources`/`source_urls` lines, with an empty line in between... (not sure why it's below the `sources` line originally, it shouldn't be)